lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -335,4 +340,64 @@ void rtc_time_set_time(uint8_t hour, uint8_t minute, uint8_t second, bool use_am
 	rtc_time_set_minute(minute);
 	rtc_time_set_second(second);
 }
-/**@}*/
+
+/*---------------------------------------------------------------------------*/
+/** @brief Gets the RTC DEC time hour value
+
+@details Requires unlocking the RTC write-protection (RTC_WPR)
```

Comment:
why? that makes no sense, and isn't listed in the ref manual I'v egot open?  (applies to all following functions as well)